provider/clickhouse.go

```diff
@@ -954,6 +955,56 @@ func (store *clickHouseOfflineStore) CreateTrainingSet(def TrainingSetDef) error
 	return nil
 }
 
+func (store *clickHouseOfflineStore) CreateTrainingTestSplit(
+	trainingSetTable string,
+	testSize float32,
+	shuffle bool,
+	randomState int,
+) (string, error) {
+	// Generate unique suffix for the view names
+	tableNameSuffix := fmt.Sprintf("%s_%d_%t_%d", trainingSetTable, int(testSize*100), shuffle, randomState)
```

Comment:
most likely, wouldn't happen but the two testSizes of 0.555 and 0.55 would generate the same name

Reply:
i drop the view afterwards so it should be fine, but it's a good point